packages/react-avatar/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Sorry for all the back and forth but this one should be deleted too (and commit with `-n` to skip the precommit checks).

Reply:
Good thing Squash merges are a thing. 😄